File: AspNetMvcUnleashed/0672329980_source/Chapter04/VB/MvcApplication1/MvcApplication1/Controllers/ProductController.vb
Public Class ProductController
    Inherits System.Web.Mvc.Controller

    private _entities As New ToyStoreDBEntities 

    Function Index() As ActionResult
        Return View(_entities.ProductSet.ToList())
    End Function

    
    Function Index2() As ActionResult
        Return View(_entities.ProductSet.ToList())
    End Function

End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter04/CS/MvcApplication1/MvcApplication1/Controllers/ProductController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using System.Web.Mvc.Ajax;
using MvcApplication1.Models;

namespace MvcApplication1.Controllers
{
    public class ProductController : Controller
    {
        private ToyStoreDBEntities _entities = new ToyStoreDBEntities();

        public ActionResult Index()
        {
            return View(_entities.ProductSet.ToList());
        }

        public ActionResult Index2()
        {
            return View(_entities.ProductSet.ToList());
        }
    }
}
